tools/update_translation.pl

```diff
@@ -187,24 +187,24 @@ sub find_pgm_files {
   }
 }
 
-$trans{ 'charset' }   = '=' if ( !defined( $trans{ 'charset' } ) );
-$trans{ 'direction' } = '=' if ( !defined( $trans{ 'direction' } ) );
-$trans{ '__mm__/__dd__/__yyyy__' } = '='
-  if ( !defined( $trans{ '__mm__/__dd__/__yyyy__' } ) );
-$trans{ '__month__ __dd__' } = '='
-  if ( !defined( $trans{ '__month__ __dd__' } ) );
-$trans{ '__month__ __dd__, __yyyy__' } = '='
-  if ( !defined( $trans{ '__month__ __dd__, __yyyy__' } ) );
-$trans{ '__month__ __yyyy__' } = '='
-  if ( !defined( $trans{ '__month__ __yyyy__' } ) );
+# $trans{ 'charset' }   = '=' if ( !defined( $trans{ 'charset' } ) );
+# $trans{ 'direction' } = '=' if ( !defined( $trans{ 'direction' } ) );
+# $trans{ '__mm__/__dd__/__yyyy__' } = '='
+#   if ( !defined( $trans{ '__mm__/__dd__/__yyyy__' } ) );
+# $trans{ '__month__ __dd__' } = '='
+#   if ( !defined( $trans{ '__month__ __dd__' } ) );
+# $trans{ '__month__ __dd__, __yyyy__' } = '='
+#   if ( !defined( $trans{ '__month__ __dd__, __yyyy__' } ) );
+# $trans{ '__month__ __yyyy__' } = '='
+#   if ( !defined( $trans{ '__month__ __yyyy__' } ) );
```

Comment:
Why are we commenting this out?  I thought it served a purpose at some point, but I don't know if it still does...